samples/bluetooth/mesh/ble_and_mesh/README.rst

```diff
@@ -0,0 +1,205 @@
+.. _bluetooth_ble_and_mesh:
+
+Bluetooth: BLE and Mesh
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® BLE and Mesh sample demonstrates how to combine Mesh and legacy BLE features in a single application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820
+
+The sample also requires a smartphone with the following apps:
+  1. Nordic Semiconductor's nRF Mesh mobile app:
+
+     * `nRF Mesh mobile app for Android`_
+     * `nRF Mesh mobile app for iOS`_
+
+  #. `nRF Connect for Mobile`_
+
+
+
+Overview
+********
+
+This sample utilizes extended advertising to allow simutanious advertising and operation for both Bluetooth® Mesh and legacy Bluetooth®.
+It combines the features of the :ref:`peripheral_lbs` and :ref:`bluetooth_mesh_light` samples into a single application.
```

Comment:
I think it's worth mentioning the Ext adv requirement here as well, as it's kind of essential for this to work. It'd also be useful to explain what the LBS is doing, and exactly what coexistence aspects are demonstrated in this sample